src/commands/repos/UpdateRepos.ts

```diff
@@ -139,28 +124,56 @@ export class UpdateRepos extends AbstractReposCommand {
         Global.isVerbose() && console.log('successfully updated', repoName);
     }
 
-    private async getTargetBranch(repo: Repository, {branch, commit, tag}: { branch: string, commit?: string, tag?: string }): Promise<string> {
+    private async checkoutTagOrBranch(repo: Repository, repoName: string, repoProperties: IRepoStatus): Promise<void> {
```

Comment:
As this could also checkout a commit, maybe rename it to just
```suggestion
    private async checkout(repo: Repository, repoName: string, repoProperties: IRepoStatus): Promise<void> {
```